### PR TITLE
EN-13240 Laundering the name twice made it invalid

### PR DESCRIPTION
**Pull request by rozap**
* Also better error messages for when stuff goes wrong;
  there is a change in core-misc to not mangle geo-import
  errors

### CODE REVIEW

File: es6-lib/errors.js [hunk 1]
```diff
@@ -252,58 +258,58 @@ class UpstreamError extends ImportError {
 
 class ConnectionError extends IOError {
   static template() {
-    return 'Failed to connect to that service: {reason}';
+    return 'Failed to connect to that service{message}';
```
Comment (kaidamasaki) [inline es6-lib/errors.js]: Why the removal of the `: `?

File: es6-lib/errors.js [hunk 2]
```diff
@@ -252,58 +258,58 @@ class UpstreamError extends ImportError {
...
 class UpsertError extends UpstreamError {
   static template() {
-    return 'Failed to upsert the dataset';
+    return 'Failed to upsert the dataset{message}';
   }
 }
 class CleanupError extends UpstreamError {
   static template() {
-    return 'Encountered an error, but encountered another error rolling back';
+    return 'Encountered an error, but encountered another error rolling back{message}';
```
Comment (kaidamasaki) [inline es6-lib/errors.js]: Spaces before the messages would probably help with readability.

File: es6-lib/errors.js (hunk 1 above)
Comment (rozap) [inline es6-lib/errors.js]: woops

File: es6-lib/errors.js (hunk 2 above)
Comment (rozap) [inline es6-lib/errors.js]: core doesn't always return a message, so i don't want there to be trailing spaces in that case.

File: es6-lib/errors.js (hunk 1 above)
Comment (rozap) [inline es6-lib/errors.js]: nice catch

**Pull request merged**